File: pyround1/852.山脉数组的峰顶索引.py
#
# @lc app=leetcode.cn id=852 lang=python3
#
# [852] 山脉数组的峰顶索引
#

# @lc code=start
class Solution:
    def peakIndexInMountainArray(self, A: List[int]) -> int:
        return (A.index(max(A)))
        
# @lc code=end

File: 852.山脉数组的峰顶索引.cpp
/*
 * @lc app=leetcode.cn id=852 lang=cpp
 *
 * [852] 山脉数组的峰顶索引
 */

// @lc code=start
class Solution {
public:
    int peakIndexInMountainArray(vector<int>& arr) {
        int max_val = *max_element(arr.begin(), arr.end());
        return find(arr.begin(), arr.end(), max_val) - arr.begin();
    }
};
// @lc code=end

